modules/op-performance-tuning-using-tektonconfig-cr.adoc

```diff
@@ -0,0 +1,63 @@
+// This module is included in the following assembly:
+//
+// *openshift_pipelines/customizing-configurations-in-the-tektonconfig-cr.adoc
+
+:_content-type: CONCEPT
+[id="op-performance-tuning-using-tektonconfig-cr_{context}"]
+= Performance tuning using TektonConfig CR
+
+You can modify the fields under the `.spec.performance` parameter in the `TektonConfig` custom resource (CR) to change high availability (HA) support and performance configurations for the {pipelines-shortname} controller.
+
+.Example: `TektonConfig` performance fields
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+  performance:
+    disable-ha: false
+    buckets: 1
+    threads-per-controller: 2
+    kube-api-qps: 5.0
+    kube-api-burst: 10
+----
+
+The fields are optional. If you set them, the {pipelines-title} Operator includes most of the fields as arguments in the `tekton-pipelines-controller` deployment under the `tekton-pipelines-controller` container. The {pipelines-shortname} Operator also updates the `buckets` field in the `config-leader-election` configuration map under the `tekton-pipelines` namespace.
+
+If you do not specify the fields, the {pipelines-shortname} Operator applies the default values for the {pipelines-shortname} controller.
```

Comment:
if you do not specify the values, operator will not update those fields. default values from pipelines-controller will be used.